app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -113,7 +113,7 @@ public void navigateToHabitViewFragment() {
         solo.clickOnView(solo.getView(R.id.add_habit_fab));
 
         // Create test habit
-        Habit testHabit = new Habit("addHabitToListTest", "Test Reason", new Date(), HABIT_ID);
+        Habit testHabit = new Habit("addHabitToListTest", "Test Reason", new Date(), HABIT_ID, new HabitEventList());
```

Comment:
Instead of newing it up in each test, can we move this to be a field that gets set in the `setUp` method instead?